root/report/ISWCsWithManyWorks.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+import PaginatedResults from '../components/PaginatedResults';
+import {WorkListRow} from '../static/scripts/common/components/WorkListEntry';
+
+import FilterLink from './FilterLink';
+import type {ReportDataT, ReportIswcT} from './types';
+
+
+const ISWCsWithManyWorks = ({
```

Comment:
There's some variable names with `ISWC` that I think we ought to rename to `Iswc` for consistency. Also in the actual file name.
```suggestion
const IswcsWithManyWorks = ({
```